packages/apputils/src/inputdialog.ts

```diff
@@ -459,3 +493,63 @@ class InputItemsDialog extends InputDialogBase<string> {
   private _list: HTMLSelectElement;
   private _editable: boolean;
 }
+
+/**
+ * Widget body for input list dialog
+ */
+class InputMultipleItemsDialog extends InputDialogBase<string> {
+  /**
+   * InputMultipleItemsDialog constructor
+   *
+   * @param options Constructor options
+   */
+  constructor(options: InputDialog.IMultipleItemsOptions) {
+    super(options.label);
+
+    let defaults = options.defaults || [];
+
+    this._list = document.createElement('select');
+
+    // Add an hidden option, otherwise the first one is selected by default
```

Comment:
This will cause issues for screen readers. Can this be avoided by explicitly setting `.selected = false` on first option or using `select.selectedIndex = -1`? I think it should not be needed but if we have to add hidden option, can we add aria-hidden?

Reply:
Agree, thanks for suggestions.
I had tried these suggestions without success some days ago, but now I can't reproduce the problem :smile: Don't know what I did...
So I removed it, and I'll fix it if necessary.